src/views/studio/MyUploadsView/AssetLine/AssetLine.tsx

```diff
@@ -47,19 +148,43 @@ const AssetLine: React.FC<AssetLineProps> = ({ isLast = false, asset }) => {
   }
 
   return (
-    <FileLineContainer isLast={isLast}>
-      {isLast ? <FileLineLastPoint /> : <FileLinePoint />}
-      <FileStatusContainer>{renderStatusIndicator(asset)}</FileStatusContainer>
-      <FileInfoContainer>
-        <FileInfoType>
-          {isVideo ? <SvgGlyphFileVideo /> : <SvgGlyphFileImage />}
-          <Text variant="body2">{fileTypeText}</Text>
-        </FileInfoType>
-        <Text variant="body2">{resolution}</Text>
-        <Text>{size}</Text>
-      </FileInfoContainer>
-      <StatusMessage variant="subtitle2">{fileStatusMessage}</StatusMessage>
-    </FileLineContainer>
+    <>
+      <MessageDialog
+        title="Different file was selected!"
+        description="We detected that you selected a different file than the one you uploaded previously. Select the same file to continue the upload or make a new transaction (0.2JOY) and upload a new version of file."
+        showDialog={showDialog}
+        variant="warning"
+        onSecondaryButtonClick={() => {
+          setShowDialog(false)
+          if (asset.parentObject.type === 'video') {
+            navigate(absoluteRoutes.studio.editVideo())
+          }
+          if (asset.parentObject.type === 'channel') {
+            navigate(absoluteRoutes.studio.editChannel())
+          }
```

Comment:
I think we got a misunderstanding on what's supposed to happen here. You redirect the user to "new video" screen when they click `Upload new`. But what should actually happen is that we create an update transaction with the new file and try to submit it, right from this screen and then start the upload. That's also what's in the diagram linked in the issue. However, maybe let's try to elevate that to separate task so we don't drag this PR for too long